src/governance-classes.cpp

```diff
@@ -156,35 +156,27 @@ void CGovernanceTriggerManager::CleanAndRemove()
     DBG( std::cout << "CGovernanceTriggerManager::CleanAndRemove: Start" << std::endl; );
     AssertLockHeld(governance.cs);
 
-    // LOOK AT THESE OBJECTS AND COMPILE A VALID LIST OF TRIGGERS
-    for(trigger_m_it it = mapTrigger.begin(); it != mapTrigger.end(); ++it) {
-        //int nNewStatus = -1;
-        CGovernanceObject* pObj = governance.FindGovernanceObject((*it).first);
-        if(!pObj) {
-            continue;
-        }
-        CSuperblock_sptr& pSuperblock = it->second;
-        if(!pSuperblock) {
-            continue;
-        }
-        // IF THIS ISN'T A TRIGGER, WHY ARE WE HERE?
-        if(pObj->GetObjectType() != GOVERNANCE_OBJECT_TRIGGER) {
-            pSuperblock->SetStatus(SEEN_OBJECT_ERROR_INVALID);
-        }
-    }
-
     // Remove triggers that are invalid or expired
     DBG( std::cout << "CGovernanceTriggerManager::CleanAndRemove: mapTrigger.size() = " << mapTrigger.size() << std::endl; );
     LogPrint("gobject", "CGovernanceTriggerManager::CleanAndRemove -- mapTrigger.size() = %d\n", mapTrigger.size());
+
     trigger_m_it it = mapTrigger.begin();
     while(it != mapTrigger.end()) {
         bool remove = false;
+        CGovernanceObject* pObj = NULL;
```

Comment:
Should we try to use nullptr in these places instead?

Reply:
Fixed